backend/authentication/views/promotions.py

```diff
@@ -6,11 +6,10 @@
 
 @api_view(["GET"])
 def list_promotions(request):
-    # Technically speaking, we are not sure at 100 % to have every promotion.
-    # In practice, it will always be OK.
-
-    promotions = [
-        x[0] % 100
-        for x in User.objects.distinct("year_of_entry").values_list("year_of_entry")
-    ]
-    return Response({"promotions": sorted(list(promotions), reverse=True)})
+    return Response(
+        {
+            "promotions": sorted(
+                set(User.objects.values_list("promotion", flat=True)), reverse=True
```

Comment:
Use `distinct` instead https://docs.djangoproject.com/en/2.2/ref/models/querysets/#distinct

Reply:
It gave me a weird SQL error… I will try again